[Doc] Better wording for 'tcp_keep_alive' option (#169)
logstash-plugins/logstash-input-tcp: docs/index.asciidoc

  * Value type is <<boolean,boolean>>
  * Default value is `false`

> Instruct the socket to use TCP keep alive. If it's `true` then the underlying socket
> will use the OS defaults settings for keep alive. If it's `false` it doesn't configure any
> keep alive setting for the underlying socket.

Reviewer: ```suggestion
Instruct the socket to use TCP keep alive. If `true` the underlying socket
will use the OS default settings for keep alive. If `false` no
keep alive settings are configured for the underlying socket.
```
